### ISSUE TITLE
Option to not show overlay?

### ISSUE THREAD

**Issue by adelnoureddine** [omnight/phonehook#72]
Is it possible to add an option to disable the overlay completely (not shown at all)?
I only use phonehook for call blocks. An option would be great.
I commented everything inside the function of the daemon's overlay.cpp, and recompiled a version which works (but a clean proper option, i.e., a toggle, would be great).

**Comment by 24mu13** [comment]
App Settings -> Show only for unknown contacts

Which is the meaning of that?
I would expect that no popup appears when checked and a saved contact calls me..
Instead it appears always!

Should we open a separate issue for that?

**Comment by mceds** [comment]
Same as @24mu13 . But I remember "show only for unknown contacts" used to work some time ago. Maybe the new Sailfish permissions system has blocked Phonehook access to the contacts database?

**Comment by sjn** [comment]
Yeah, this would be a welcome addition. The "Show only for unknown contacts" doesn't work atm. (SailfishOS 4.3.0.12)
